modules/structs/miscellaneous.go

```diff
@@ -21,7 +21,7 @@ type MarkupOption struct {
 	//
 	// in: body
 	Text string
-	// Mode to render (comment, gfm, markdown, file)
+	// Mode to render (markdown, comment, wiki, file)
```

Comment:
So this will become a break change?

Reply:
No, no breaking.

The "gfm" support is still there, and the "gfm" tests still pass.

See the code.